### ISSUE TITLE
Usage without Sanity

### ISSUE THREAD

**Issue by kb-0912** [tinloof/medusa-dtc-starter-munchies#36]
Is there any option to usage this starter without Sanity?

**Comment by diboune** [comment]
We don't have another version for this exact starter without `Sanity`, but you can always strip this down into your use case, or check other [Medusa starters](https://medusajs.com/starters/)

**Comment by NomadXD** [comment]
@diboune Any plans to release a version without Sanity? With sanity this template is barely usable as a starter.

**Comment by daic11** [comment]
Yes please update, want to use this starter but without Sanity @diboune
